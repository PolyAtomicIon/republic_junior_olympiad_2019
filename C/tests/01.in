13 12
408 992 941 993 796 464 349 231 297 403 290 685 368 