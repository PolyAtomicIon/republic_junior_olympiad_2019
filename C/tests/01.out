408 5888 44669 238173 1004118 3566218 11096386 31057272 79675831 190000085 425715151 903871636 830967542 